pipelines/sqre/infra/build_sciplatlab.groovy

```diff
@@ -58,13 +58,42 @@ notify.wrap {
         conn.doOutput = true
         conn.outputStream << json.toPrettyString()
 
-        println("responseCode: ${conn.responseCode}")
-        def text = conn.getInputStream().getText()
-        println("response: ${text}")
-      }
-    }
-  } // run
+        assert (conn.responseCode == 204) : "API dispatch failed: ${conn.responseCode}|${conn.getInputStream().getText()}"
+      } // openConnection().with
+    } // stage
+
+    stage('poll for job completion at GHA') {
+      // We presume that GH Actions will continue to list most recent runs
+      // at the top.  Poll the build.yaml runs for the first result.
+
+      // I think we always want the prod branch.
 
+      def url = new URL("https://api.github.com/repos/lsst-sqre/sciplat-lab/actions/workflows/build.yaml/runs?per_page=1&branch=prod")
+
+      def status = ""
+      def conclusion = ""
+      def loop_idx = 0
+      while (status != "completed") {
+        loop_idx += 1
+        if (loop_idx > 1) {
+          sleep(10 * 1000)  // Good old sleep 10 (but not the first time)
+        }
```

Comment:
Maybe abort if `loop_idx` gets too big (GHA is stuck for some reason)?

Reply:
OK, I set it to 2 hours.  I should probably also put a timeout in the action itself.  It takes about 25 minutes, usually.  But it's also very dependent on how the conda/mamba solve goes within the container build.